### PR TITLE
LRSD-4238 [Testray] Implemented Subtask Case Results update on complete

### PR DESCRIPTION
**Pull request by JoMaAlves**

### CODE REVIEW

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

**Review by joseabelenda — CHANGES_REQUESTED** [review]

File: workspaces/liferay-testray-workspace/modules/testray-rest-impl/src/main/java/com/liferay/testray/rest/internal/util/TestrayUtil.java [hunk 1]
```diff
@@ -87,4 +87,21 @@ public static List<Map<String, Object>> runSQL(
 		return list;
 	}
 
+	public static int updateSQL(String sql, List<Object> params)
```
[the remark this answers is not in the document]
Comment (JoMaAlves) [reply workspaces/liferay-testray-workspace/modules/testray-rest-impl/src/main/java/com/liferay/testray/rest/internal/util/TestrayUtil.java:90]: Done!

File: workspaces/liferay-testray-workspace/modules/testray-rest-impl/rest-openapi.yaml [hunk 2]
```diff
@@ -38,6 +38,26 @@ components:
                     format: int64
                     type: integer
             type: object
+        TestrayCaseResultTestFlowUpdate:
+            properties:
+                amount:
```
[the remark this answers is not in the document]
Comment (JoMaAlves) [reply workspaces/liferay-testray-workspace/modules/testray-rest-impl/rest-openapi.yaml:43]: Done!

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by joseabelenda** [comment]
resent:https://github.com/brianchandotcom/liferay-portal/pull/149972

**Pull request closed without merging**